docs/en/Community-Articles/2024-05-07-using-fluent-validation-with-abp/post.md

```diff
@@ -0,0 +1,95 @@
+# Using FluentValidation with ABP
+
+## What is Validation? Why Do We Need to Validate?
+Validation is checking whether a data is valid or not. We can liken validations to the cell membrane of our application. Just as the cell does not want to let in harmful substances, we do not want to add erroneous data to the database, which is critical to our application. Validations allow data that follows the rules to reach the database. Data that does not comply with the rules will not access the database at all, and the operation will fail.
+
+![validation](./images/validation.png)
+
+Validations make your application run more efficiently because they are not connected to the database.  Validations should be applied in different layers of applications. Validations are applied on the UI side. Validations should also be applied on the backend side and database to prevent malicious users from passing these validations. Fluent validation is often used to create validations on the backend side.
```

Comment:
Validations actually are not just tied to database point and they are used to validate your application's internal state. So, in this sentence, it's misleading and pointing out that it's main use is to keep the database in the correct state rather than keeping the application in the correct state. So, I think we should improve this sentence. Here is an example, sentence that came to my mind:

```suggestion
Validations make your application run more efficiently as they are not directly tied to the database. They should be implemented across various layers of the application, including the UI, backend, and database, to prevent malicious users from bypassing these checks. Fluent validation is frequently employed for creating backend validations.
```

You can change this sentence according to your own sentences, but keep that in mind, you should not just specify the validations are just for keeping the database in a correct state, rather it's used to keep all of your business rules in the correct order with the correct state (internal state of your application).